src/AutoscaleManager/NodeManager/NodeManagerService.cs

```diff
@@ -166,7 +181,7 @@ private async Task RemoveScaledInNodesAsync(CancellationToken cancellationToken)
                         continue;
                     }
 
-                    if (node.NodeStatus == NodeStatus.Down)
+                    if (node.NodeStatus == NodeStatus.Down && nodeTypesToManage.Contains(node.NodeType))

```

Comment:
this comparison should be case invariant, and you need to strip out whitespaces as people can add whitespaces anywhere in the comma separated list.